drivers/overlay/overlay.go

```diff
@@ -292,6 +292,15 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 		backingFs = fsName
 	}
 
+	runhome := filepath.Join(options.RunRoot, filepath.Base(home))
+	supported, err := SupportsNativeOverlay(home, runhome)
```

Comment:
should we check for `err`?

Reply:
I think we should. All the errors that `SupportsNativeOverlay` reports are rather low level issues that could indicate something is up with your file system or storage, so I think these shouldn't be ignored.